t/t2204-job-info.t

```diff
@@ -16,29 +16,33 @@ if test "$TEST_LONG" = "t"; then
     test_set_prereq LONGTEST
 fi
 
+fj_wait_event() {
```

Comment:
Assuming this change doesn't become it's own PR (per the slack discussion), I'd recommend making it it's own commit in this PR.

Reply:
I did not mean for this change to land here... and there is more fixup to actually make it work, alternate PR coming.

Reply:
I think I got this removed from the PR, and moved various other test fixes over to #2849.  Hopefully that still lets this one pass, we shall see.